modules/git/repo_tag_test.go

```diff
@@ -26,11 +26,14 @@ func TestRepository_GetTags(t *testing.T) {
 		assert.NoError(t, err)
 		return
 	}
-	assert.Len(t, tags, 1)
+	assert.Len(t, tags, 2)
 	assert.Equal(t, len(tags), total)
-	assert.EqualValues(t, "test", tags[0].Name)
-	assert.EqualValues(t, "3ad28a9149a2864384548f3d17ed7f38014c9e8a", tags[0].ID.String())
+	assert.EqualValues(t, "signed-tag", tags[0].Name)
```

Comment:
Is this always element 0 now that there are two tags?

Reply:
Haven't checked the code but generally order depends on `tag.sort` git config.

Reply:
It sorts by date so it should be stable ordering:

```
$ GIT_DIR=modules/git/tests/repos/repo1_bare git for-each-ref --sort '-*creatordate' refs/tags
36f97d9a96457e2bab511db30fe2db03893ebc64 tag	refs/tags/signed-tag
3ad28a9149a2864384548f3d17ed7f38014c9e8a tag	refs/tags/test
```